docs/cookbook/languages/rust.md

```diff
@@ -269,5 +269,5 @@ runtime-packages = ["libiconv", "gcc"]
 [custom-toolchains]: https://github.com/zmitchell/rust-toolchains
 [esp32]: https://www.espressif.com/en/products/socs/esp32
 [risc-v]: https://en.wikipedia.org/wiki/RISC-V
-[build-concept]: ../../concepts/manifest-builds.md
+[build-concept]: ../../concepts/builds.md
```

Comment:
(unrelated line)
suggestion blocking: it looks like this link also needs to be changed in c.md to point at builds.md rather than manifest-builds.md. I think that was dropped in a bad merge